ubuntu-advantage

```diff
@@ -2,13 +2,14 @@
 
 SCRIPTNAME=$(basename "$0")
 
-SERIES=`lsb_release -cs`
+SERIES=$(lsb_release -cs)
```

Comment:
Please keep using backslashes. Even though Ubuntu's /bin/sh (-> dash) supports $(), the rest of the script uses backslashes.